scmerlin.sh

```diff
@@ -308,8 +308,8 @@ GetTemperatureString()
     then printf "${REDct}*Unknown*${CLRct}"
     fi
     if ! echo "$1" | grep -qE "^[0-9].*"
-    then printf "${REDct}%s${CLRct}" "$theTemptrVal"
-    else printf "${GRNct}%s°C${CLRct}" "$theTemptrVal"
+    then printf "${REDct}%s${CLRct}" "$1"
+    else printf "${GRNct}%s°C${CLRct}" "$1"
```

Comment:
This was actually correct and as intended; the variable `theTemptrVal` has already been set in a previous function call and passed as the argument just to check if we have a temperature value or some error message. But,  I can see how it may appear wrong and confusing unless you follow the execution flow.